apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -50,69 +50,72 @@ function WpcomNux() {
 
 	const dismissWpcomNux = () => setWpcomNuxStatus( { isNuxEnabled: false } );
 
+	/* @TODO: the copy, images, and slides will eventually be the same for all sites. `isGutenboarding` is only needed right now to show the Privacy slide */
 	const isGutenboarding = !! window.calypsoifyGutenberg?.isGutenboarding;
-	const welcomeHeading = isGutenboarding
-		? __( 'Welcome to your website' )
-		: __( 'Welcome to the WordPress editor' );
 
 	return (
 		<Guide
 			className="wpcom-block-editor-nux"
-			contentLabel={ welcomeHeading }
-			finishButtonText={ __( 'Get started' ) }
+			contentLabel={ __( 'Welcome to your website', 'full-site-editing' ) }
+			finishButtonText={ __( 'Get started', 'full-site-editing' ) }
 			onFinish={ dismissWpcomNux }
 		>
-			<NuxPage
-				heading={ welcomeHeading }
-				description={
-					isGutenboarding
-						? __(
-								'Edit your homepage, add the pages you need, and change your site’s look and feel.'
-						  )
-						: __( 'Create and edit site pages, and customize the look and feel of each page.' )
-				}
-				imgSrc={ editorImage }
-				alignBottom
-			/>
-
-			<NuxPage
-				heading={
-					isGutenboarding
-						? __( 'Customize your content' )
-						: __( 'Create pages and add your content' )
-				}
-				description={
-					isGutenboarding
-						? __( 'Start with an existing layout and make it your own.' )
-						: __(
-								'Create and rearrange your site pages. Customize your site navigation menus so your visitors can explore your site.'
-						  )
-				}
-				imgSrc={ blockImage }
-			/>
-
-			<NuxPage
-				heading={ __( 'Add (almost) anything' ) }
-				description={ __(
-					'Insert text, photos, forms, Yelp reviews, testimonials, maps, and more. Rearrange the blocks on your page until they’re just right.'
-				) }
-				imgSrc={ blockPickerImage }
-			/>
-
-			{ /* @TODO: hide for sites that are already public
-			<NuxPage
-				heading={ __( "Private until you're ready to launch" ) }
-				description={ __(
-					"Your site remains private until you're ready to launch. Take your time and make as many changes as you need until it's ready to share with the world."
-				) }
-				imgSrc={ privateImage }
-				alignBottom
-			/>
-			*/ }
+			{ getWpcomNuxPages( isGutenboarding ).map( ( nuxPage ) => (
+				<NuxPage { ...nuxPage } />
```

Comment:
This doesn't complain about missing `key`?

Reply:
We test on wp-admin where these warnings don't seem to show up.

Reply:
Added heading as the key in https://github.com/Automattic/wp-calypso/pull/41966/commits/c0af4f691f73380ab3c336f3fdb462a459a55758

Reply:
> This doesn't complain about missing key?

Doh, forgot the key. Thanks for adding.